int64 id
string name
Point pos
Quaternion rot
